tests/php/ORM/DataQueryTest/AddsToQuery.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace SilverStripe\ORM\Tests\DataQueryTest;
+
+use SilverStripe\Dev\TestOnly;
+use SilverStripe\ORM\DataObject;
+
+class AddsToQuery extends DataObject implements TestOnly
```

Comment:
```suggestion
class DataObjectAddsToQuery extends DataObject implements TestOnly
```
It's a bit of a randomly named class, just make it more obvious what this is, also consistent with use of `DBFieldAddstoQuery`

Reply:
Done